java/mapping/types.js

```diff
@@ -134,6 +136,10 @@
     "TestResultContainer": {
         name: () => `TestResultContainer`,
         get: (target, key) => key.includes('get') ? `${target}.${key}` : simpleGetter(target, key)
+    },
+    "rect": {
+        name: () => 'Rect',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
        name: () => "Rect",
```